src/thread_task.rs

```diff
@@ -1130,12 +1130,12 @@ pub unsafe fn rav1d_worker_task(c: &Rav1dContext, task_thread: Arc<Rav1dTaskCont
                                 if error_0 == 0 {
                                     rav1d_cdf_thread_update(
                                         frame_hdr,
-                                        &mut *f.out_cdf.data.cdf,
+                                        &mut f.out_cdf.cdf().unwrap().try_write().unwrap(),
```

Comment:
The `cdf().unwrap()` is redundant with the `refresh_context != 0` above.

Reply:
`cdf_write()` now tries to lock, so we need the unwrap.